browser/src/map/handler/Map.Keyboard.js

```diff
@@ -671,6 +671,11 @@ L.Map.Keyboard = L.Handler.extend({
 			return true;
 		}
 
+		// Control + F3, ignore Autotext dialog
+		if (this._isCtrlKey(e) && e.keyCode === this.keyCodes.F3) {
```

Comment:
I see it is merged already, but I have some comment for that:
- if you need to add/modify any keyboard shortcut - please add it into https://github.com/CollaboraOnline/online/blob/master/browser/src/map/handler/Map.KeyboardShortcuts.ts#L161
- it allows deterministic, easily auditable way to define shortcuts and overwrites for different languages
- adding more ifs to Map.Keyboard makes it harder to maintain and we had many regressions due to not easy to spot connections in the past

Reply:
@eszkadev thank you for review. Let me fix it : )

Reply:
Done: https://github.com/CollaboraOnline/online/pull/11069